src/shell-interface.h

```diff
@@ -142,7 +142,7 @@ struct ShellExternalInterface : ModuleInstance::ExternalInterface {
       std::cout << "exit()\n";
       throw ExitException();
     }
-    std::cout << "callImport " << import->name.str << "\n";
```

Comment:
Sounds good.  Seems like perhaps LLVM_UNREACHABLE() might have been the wrong thing to use here.